### ISSUE TITLE
Pendulum.in_timezone doesn't like datetime.timezone.utc

### ISSUE THREAD

**Issue by numberoverzero** [sdispater/pendulum#97]
This could just be me misunderstanding what the valid inputs are for `in_timezone` but I was very surprised that `datetime.timezone.utc` worked for `Pendulum.instance` but not `Pendulum.in_timezone`:

```python
import datetime
import pendulum

dt_utc = datetime.timezone.utc
dt = datetime.datetime(2017, 3, 5, 3, 1, 46, 425154, tzinfo=dt_utc)

# This has no problem with dt_utc
p = pendulum.instance(dt)

# raises ValueError: Unsupported timezone UTC+00:00
p.in_timezone(dt_utc)
```

**Comment by sdispater** [comment]
Which version of `pendulum` are you using?

This issue has been fixed in `1.0.2`.

**Comment by numberoverzero** [comment]
You are correct! I was still on 0.6.5.  Sorry about that!
